content/docs/glossary/claimable-balance.mdx

```diff
@@ -49,29 +49,114 @@ This operation will load the ClaimableBalanceEntry that corresponds to the Balan
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+)
+
+func main() {
+	client := sdk.DefaultTestNetClient
+
+	// Suppose that these accounts exist and are funded accordingly:
+	A := "SCZANGBA5YHTNYVVV4C3U252E2B6P6F5T3U6MM63WBSBZATAQI3EBTQ4"
+	B := "GA2C5RFPE6GCKMY3US5PAB6UZLKIGSPIUKSLRB6Q723BM2OARMDUYEJ5"
+
+	// NOTE: Error checks are omitted for brevity; always check return values!
+
+	// Load the corresponding account for A.
+	aKeys := keypair.MustParseFull(A)
+	aAccount, err := client.AccountDetail(sdk.AccountRequest{
+		AccountID: aKeys.Address(),
+	})
+
+	// Create a claimable balance with our two above-described conditions.
+	soon := time.Now().Add(time.Second * 60)
+	bCanClaim := txnbuild.BeforeRelativeTimePredicate(60)
+	aCanReclaim := txnbuild.NotPredicate(
+		txnbuild.BeforeAbsoluteTimePredicate(soon.Unix()),
+	)
+
+	claimants := []txnbuild.Claimant{
+		txnbuild.NewClaimant(B, &bCanClaim),
+		txnbuild.NewClaimant(aKeys.Address(), &aCanReclaim),
+	}
+
+	// Create the operation and submit it in a transaction.
+	claimableBalanceEntry := &txnbuild.CreateClaimableBalance{
+		Destinations: claimants,
+		Asset:        txnbuild.NativeAsset{},
+		Amount:       "420",
+	}
+
+	// Build, sign, and submit the transaction
+	tx, err := txnbuild.NewTransaction(
+		txnbuild.TransactionParams{
+			SourceAccount:        &aAccount,
+			IncrementSequenceNum: true,
+			BaseFee:              txnbuild.MinBaseFee,
+			Timebounds:           txnbuild.NewInfiniteTimeout(), // Use a real timeout in production!
+			Operations:           []txnbuild.Operation{claimableBalanceEntry},
+		},
+	)
+	tx, err = tx.Sign(network.TestNetworkPassphrase, aKeys)
+	txResp, err := client.SubmitTransaction(tx)
+
+	fmt.Println("Claimable balance created!")
+}
+```
 
-```cpp
-// In this scenario, AccountB has not been created yet, but we know the public key
-Claimants = [Claimant(AccountA, CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(100))), Claimant(AccountB, CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(50))]
-
-BalanceID = AccountA.CreateClaimableBalance(100, XLM, Claimants)
-
-// Now one of two successful claims can happen -
+</CodeExample>
 
-// Before time=50, accountB can be created and claim the balance
-CreateAccount(AccountB)
-AccountB.ClaimClaimableBalance(BalanceID)
+At this point, the claimable balance exists in the ledger. We can now claim it
+by its Balance ID. This can be acquired in two ways:
+ 1. the submitter of the entry (Account A in this case) parses the XDR of the transaction result's operations, **or**
+ 2. someone queries the list of claimable balances (filtering accordingly, if necessary)
 
-// OR
+<CodeExample>
 
-// After time=100, AccountA can claim the balance
-AccountA.ClaimClaimableBalance(BalanceID)
+```go
+// Suppose `txResp` comes from the transaction submission above.
+var txResult xdr.TransactionResult
+err = xdr.SafeUnmarshalBase64(txResp.ResultXdr, &txResult)
+results, ok := txResult.OperationResults()
```

Comment:
since you're not checking `ok`, you can use `txResult.MustOperationResults()`

Reply:
Comments

Reply:
Comments

Reply:
0x1c8f8334Ff5FB5De670F06B554Ed1cD734902ED5

Reply:
![Uploading 95004508...]

Reply:
@tamirms exclusion of error-checking was a bit intentional (see [line 79 above](https://github.com/stellar/new-docs/pull/251/files#diff-8e23523e5c6850346fe757c291cf5732R79)): if a reader copy-pastes the examples and tries to compile, they'll get errors because `err` and `ok` are unchecked. 

Changing this is obviously not a big deal, but are we sure we want examples to be *completely* plug-and-play? Just musing from a "teaching" perspective.

Reply:
I think it's good to model error checking if not too verbose. People will cut and paste this stuff, even into production (trust me). If you feel it will be too long-winded, you can follow the same convention as in the godoc, e.g. https://godoc.org/github.com/stellar/go/txnbuild#example-AccountMerge. We just do a
```
check(err)
```
to indicate error checking is needed.

Reply:
Fair enough! Addressed :+1: